.github/workflows/build.yml

```
@@ -25,6 +27,14 @@ on:
         description: Upload built packages to artifactory
         type: boolean
         default: false
+      build_dependants:
+        description: Parse conan dependency tree and build not just the requested package, but also all dependants of it
```

```suggestion
        description: Parse conan dependency tree and build not just the requested package, but also all dependents of it
```

Had to Google that: dependant (singular), dependents (plural) :exploding_head: